Tests/Cell.py

```diff
@@ -1126,6 +1165,35 @@ def test_execute_mdx_dataframe(self):
             self.total_value,
             sum(values))
 
+    @skip_if_no_pandas
+    def test_execute_mdx_dataframe_row_only(self):
```

Comment:
Should be `test_execute_mdx_dataframe_column_only`

Reply:
Oh yeah! got confused while fixing it.